test/forms/external-data.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0"?>
```

Comment:
This file can be removed I think, since it is superseded by v2 above.

Reply:
file removed